File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/Employee.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class Employee {
    @JsonProperty("FullName")
    public String fullName;
    @JsonProperty("IsIndustryAdmin")
    public Boolean isIndustryAdmin;
    @JsonProperty("IsManager")
    public Boolean isManager;
    @JsonProperty("IsOwner")
    public Boolean isOwner;
    @JsonProperty("License")
    public EmployeeLicense license;
    public static class EmployeeLicense {
    @JsonProperty("EmployeeLicenseNumber")
    public String employeeLicenseNumber;
    @JsonProperty("EndDate")
    public String endDate;
    @JsonProperty("LicenseType")
    public String licenseType;
    @JsonProperty("Number")
    public String number;
    @JsonProperty("StartDate")
    public String startDate;
}
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/Employee.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class Employee(
    @JsonProperty("FullName")
    val fullName: String? = null,
    @JsonProperty("IsIndustryAdmin")
    val isIndustryAdmin: Boolean? = null,
    @JsonProperty("IsManager")
    val isManager: Boolean? = null,
    @JsonProperty("IsOwner")
    val isOwner: Boolean? = null,
    @JsonProperty("License")
    val license: EmployeeLicense? = null
)
